### ISSUE TITLE
Bar chart vertical spacing

### ISSUE THREAD

**Issue by arjhunAtcodingmart** [apexcharts/apexcharts.js#1494]
**What's Currently Missing**
Currently we cannot able to reduce or increase the spacing between the bar's in the bar chart.

**What's Needed**
So it will be helpful if have a offset option to increase or decrease the vertical spacing between the bar's in the bar chart.

**Comment by junedchhipa** [comment]
Have you tried setting 
```javascript
plotOptions: {
  bar: {
    barHeight: '80%'
  }
}
```
Adjust the percentage and the bars will be adjusted to the available space.

**Comment by arjhunAtcodingmart** [comment]
Yes, I have tried this option, but not able to control the x-axis spacing between the two bars.

**Comment by arjhunAtcodingmart** [comment]
> @junedchhipa Yes, I have tried this option, but not able to control the x-axis spacing between the two bars.
> 
> I am using a vertical bar and need to decrease the spacing between the two bars.
> 
> ![image](https://user-images.githubusercontent.com/44165466/79419083-bb92ab80-7fd3-11ea-9250-997aae5f9535.png)

**Comment by tungvodoi** [comment]
> @junedchhipa Yes, I have tried this option, but not able to control the x-axis spacing between the two bars.
> 
> I am using a vertical bar and need to decrease the spacing between the two bars.
> 
> ![image](https://user-images.githubusercontent.com/44165466/79419083-bb92ab80-7fd3-11ea-9250-997aae5f9535.png)

Did you resolve this issue, I've meet the same problem
